pkg/resources/management.cattle.io/v3/projectroletemplatebinding/ProjectRoleTemplateBinding.md

```diff
@@ -2,24 +2,34 @@
 
 ### Escalation Prevention
 
-Users can only create/update ProjectRoleTemplateBindings with rights less than or equal to those they currently possess. This is to prevent privilege escalation. 
+Users can only create/update ProjectRoleTemplateBindings with rights less than or equal to those they currently possess.
+This is to prevent privilege escalation.
 
 ### Invalid Fields - Create
 
-Users cannot create ProjectRoleTemplateBindings which violate the following constraints:
-- Either a user subject (through `UserName` or `UserPrincipalName`) or a group subject (through `GroupName` or `GroupPrincipalName`) must be specified; both a user subject and a group subject cannot be specified
+Users cannot create ProjectRoleTemplateBindings that violate the following constraints:
+
+- Either a user subject (through `UserName` or `UserPrincipalName`), or a group subject (through `GroupName`
+  or `GroupPrincipalName`), or a service account subject (through `ServiceAccount`) must be specified. Exactly one
+  subject type of the three must be provided.
 - `ProjectName` must be specified
 - The roleTemplate indicated in `RoleTemplateName` must be:
-  - Valid (i.e. is an existing `roleTemplate` object in the `management.cattle.io/v3` API group)
-  - Not locked (i.e. `roleTemplate.Locked` must be `false`)
+    - Provided as a non-empty value
+    - Valid (there must exist a `roleTemplate` object of given name in the `management.cattle.io/v3` API group)
+    - Not locked (`roleTemplate.Locked` must be `false`)
+    - Associated with its appropriate context (`roleTemplate.Context` must be equal to "project")
 
 ### Invalid Fields - Update
 
 Users cannot update the following fields after creation:
+
 - RoleTemplateName
 - ProjectName
+- ServiceAccount
```

Comment:
It's a bit odd to me that this one is in the "can't update" section rather than in the "can update if not set but not changeable after" section with the other subject fields. I think I understand why, but would you mind clarifying why so that I'm sure I've got the idea?

Reply:
Discussed offline, will keep existing wording.